javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,190 @@
+
+  const formTitle = document.createElement("h1");
+  formTitle.textContent = "New recipe";
+  newForm.appendChild(formTitle);
+
+  const formName = document.createElement("label");
+  formName.textContent = "Add name of dish";
+  const nameInput = document.createElement("input");
+  nameInput.id = "recipe-title";
+  formName.appendChild(nameInput);
+  newForm.appendChild(formName);
+
+  newForm.appendChild(document.createElement("br"));
+
+  const formImg = document.createElement("label");
+  formImg.textContent = "Add image of dish";
+  const imgInput = document.createElement("input");
+  imgInput.id = "recipe-image";
+  formImg.appendChild(imgInput);
+  newForm.appendChild(formImg);
+
+  newForm.appendChild(document.createElement("br"));
+
+  const labelIngredients = document.createElement("label");
+  labelIngredients.textContent = "Ingredients (min 5):";
+  newForm.appendChild(document.createElement("br"));
+  newForm.appendChild(labelIngredients);
+
+  const ingredientsContainer = document.createElement("div");
+  ingredientsContainer.id = "ingredients-container";
+  newForm.appendChild(ingredientsContainer);
+
+  const addIngredientButton = document.createElement("button");
+  addIngredientButton.type = "button";
+  addIngredientButton.textContent = "Add ingredient";
+  addIngredientButton.addEventListener(`click`, addFormForIngredient);
+  
+  newForm.appendChild(addIngredientButton);
+  newForm.appendChild(document.createElement("br"));
+
+  function addFormForIngredient() {
+    const ingredientInput = document.createElement("div");
+    ingredientInput.classList.add("ingredient");
+
+    const nameInput = document.createElement("input");
+    nameInput.classList.add("ingredient-name");
+    nameInput.placeholder = "Ingredient name";
+
+    const amountInput = document.createElement("input");
+    amountInput.classList.add("ingredient-amount");
+    amountInput.placeholder = "Amount";
+
+    ingredientInput.appendChild(nameInput);
+    ingredientInput.appendChild(amountInput);
+    ingredientsContainer.appendChild(ingredientInput);
+  }
+
+  for (let i = 0; i < 5; i++) {
+    addFormForIngredient();
+  }
+
+  newForm.appendChild(document.createElement("br"));
+
+  const formDescription = document.createElement("label");
+  formDescription.textContent = "Description:";
+  const descriptionInput = document.createElement("input");
+  descriptionInput.id = "recipe-description";
+  formDescription.appendChild(descriptionInput);
+  newForm.appendChild(formDescription);
+  
+  newForm.appendChild(document.createElement("br"));
+
+  const addRecipe = document.createElement("button");
+  addRecipe.textContent = "Add recipe";
+  addRecipe.type = "button";
+
+  addRecipe.addEventListener(`click`, function (event) {
+    event.preventDefault();
+
+    const title = nameInput.value.trim();
+    const image = imgInput.value.trim();
+    const description = descriptionInput.value.trim();
+
+    if (!title || !image || !description) {
```

Comment:
Good job checking if anything is not filled in, plus having value.trim() so no empty spaces.